.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Motivation and Resolution
+...
+
+### RPC version
```

Comment:
should this be mandatory ? The change could also come from something different than an roc version change

Reply:
True, could add next to it something like "if applicable"

Reply:
[cd7a8ac](https://github.com/0xs34n/starknet.js/pull/302/commits/cd7a8ac2e049595d0bc78a5c4cd7c3cd0582c8df)